@here/harp-datasource-protocol/lib/TechniqueParams.ts

```diff
@@ -699,12 +725,31 @@ export interface MarkerTechniqueParams extends BaseTechniqueParams {
     wrappingMode?: DynamicProperty<"None" | "Character" | "Word">;
     /**
      * Text position regarding the baseline.
+     *
+     * @note The [[placements]] attribute may override the alignment settings when using
+     * alternative placements algorithm.
      */
     hAlignment?: DynamicProperty<"Left" | "Center" | "Right">;
     /**
      * Text position inside a line.
+     *
+     * @note The [[placements]] attribute may supersede it, when using alternative placement
+     * algorithm.
```

Comment:
I would remove "when using alternative placement algorithm". It doesn't mean anything for a client reading this API, and it doesn't explain anything additional for the internal developer.
I would just put: "@note The [[placements]] attribute may override the alignment settings"

Reply:
Ok.

Reply:
Ok.